apps/prairielearn/assets/scripts/lib/editAssessmentQuestionsTable.ts

```diff
@@ -0,0 +1,374 @@
+        <div ${question.is_alternative_group ? 'class="ml-3"' : ''}>
+          <button
+            class="btn btn-xs btn-secondary question-up-arrow-button"
+            type="button"
+            data-zone-number="${zoneIndex}"
+            data-question-number="${questionIndex}"
+            data-alternative-number="${alternativeIndex}"
+            ${zoneIndex === 0 && questionIndex === 0 && alternativeIndex === undefined
+              ? 'disabled'
+              : ''}
+          >
+            <i class="fa fa-arrow-up" aria-hidden="true"></i>
+          </button>
+        </div>
+        <div>
+          <button
+            class="btn btn-xs btn-secondary question-down-arrow-button"
+            type="button"
+            data-zone-number="${zoneIndex}"
+            data-question-number="${questionIndex}"
+            data-alternative-number="${alternativeIndex}"
+          >
+            <i class="fa fa-arrow-down" aria-hidden="true"></i>
+          </button>
+        </div>
+      </td>
+      <td class="editButtonCell align-content-center">
+        <button
+          class="btn btn-sm btn-secondary editButton"
+          type="button"
+          data-zone-number="${zoneIndex}"
+          data-question-number="${questionIndex}"
+          data-zone-number="${zoneIndex}"
+          data-question-number="${questionIndex}"
+          data-alternative-number="${alternativeIndex}"
+          data-toggle="modal"
+          data-target="editQuestionModal"
+        >
+          <i class="fa fa-edit" aria-hidden="true"></i>
+        </button>
+      </td>
+      <td class="deleteButtonCell align-content-center">
+        <button
+          class="btn btn-sm btn-danger deleteQuestionButton"
+          type="button"
+          data-zone-number="${zoneIndex}"
+          data-question-number="${questionIndex}"
+          data-alternative-number="${alternativeIndex}"
+          data-toggle="modal"
+          data-target="deleteQuestionModal"
+        >
+          <i class="fa fa-trash" aria-hidden="true"></i>
+        </button>
+      </td>
+      <td class="align-content-center" colspan="${question.is_alternative_group ? '1' : '2'}">
+        ${!question.is_alternative_group
+          ? `${question.alternative_group_number}.`
+          : html`
+              <span class="ml-3">
+                ${question.alternative_group_number}.${question.number_in_alternative_group}.
+              </span>
+            `}
+        ${question.title}
+      </td>
+      <td class="align-content-center">${question.qid}</td>
+      <td>${TopicBadge(question.topic)}</td>
+      <td>${TagBadgeList(question.tags)}</td>
+      <td class="align-content-center">
+        ${maxPoints({
+          max_auto_points,
+          max_manual_points,
+          points_list,
+          init_points,
+          assessmentType,
+        })}
+      </td>
+      <td class="align-content-center">${max_manual_points || '—'}</td>
```

Comment:
I believe we still wanted to have popovers here so we can edit points without the entire `editQuestionModal` similar to the `assessmentInstance` page. I can add those in either here or in a subsequent PR.

Reply:
I think a subsequence PR is fine, let's try to freeze features here before things grow out of control. Consider making an issue for yourself to track overarching goals, things to implement in followup PRs, and so on.